frontend/src/routes/(frontend)/[accountId]/[statusId]/index.tsx

```diff
@@ -36,11 +36,12 @@ export default component$(() => {
 			<div class="bg-wildebeest-700 p-4">
 				{/* Account Card */}
 				<div class="flex">
-					<Avatar src={status.account.avatar} />
+					<Avatar primary={status.account} secondary={null} />
```

Comment:
wouldn't it better to make `secondary` optional instead of having to specifically pass `null`?

Reply:
I generally like to be explicit if possible. I find that optional params are best only on public APIs.
Here I am reminding us that perhaps we should consider reblogs in individual statuses??
I wasn't sure.

Reply:
ah I see, I'm fine with keeping it as is 🙂:+1: